src/Wallabag/CoreBundle/Form/Type/EntryFilterType.php

```diff
@@ -91,6 +92,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label' => 'entry.filters.domain_label',
             ])
             ->add('httpStatus', TextFilterType::class, [
+                'apply_filter' => function (QueryInterface $filterQuery, $field, $values) {
+                    $value = $values['value'];
+                    if (false === array_key_exists($value, Response::$statusTexts)) {
+                        return;
+                    }
+
+                    $paramName = sprintf('%s', str_replace('.', '_', $field));
```

Comment:
As explained in the bundle doc, we need to replace the dot https://github.com/lexik/LexikFormFilterBundle/blob/master/Resources/doc/working-with-the-bundle.md#ii-filter-customization